### DISCUSSION TITLE
关于sidebar

### DISCUSSION THREAD

**Discussion by zuihou** [vuepress-theme-hope/vuepress-theme-hope#3022] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8500410/232052176-8344b8a8-cc93-4fa0-baa0-18ed6f0526bd.png)

左侧菜单如何默认 居左 ，并且显示 收缩按钮 呢？

```
  "dependencies": {
    "@vuepress/client": "2.0.0-beta.61",
    "@vuepress/plugin-docsearch": "2.0.0-beta.61",
    "@vuepress/utils": "2.0.0-beta.61",
    "vue": "^3.2.47",
    "vuepress": "2.0.0-beta.61",
    "vuepress-theme-hope": "2.0.0-beta.201"
  },
```

**Comment by Mister-Hope** [comment, marked as answer]
- https://theme-hope.vuejs.press/zh/guide/interface/responsive.html
- https://theme-hope.vuejs.press/zh/guide/layout/sidebar.html
